docs/source/introduction.rst

```diff
@@ -7,60 +8,108 @@ This section covers installation of pyChapel and required software packages,
+
 Installation
 ~~~~~~~~~~~~
 
-The steps for installing pyChapel and the software packages it depends upon is
-fairly straightforward. The following provides a setup guide on a Ubuntu-linux system.
+The steps for installing pyChapel and the software packages it depends upon are
+fairly straightforward. The following provides a setup guide on a Ubuntu-16.04
+host system.
+
+The pyChapel Python module itself is available from PyPi.
+Make sure you are using the Python2 version of Pip and any other Python utilities.
+We recommend using Pip's "user install" option as shown here, but those familiar
+with other Python2 virtual environments may adapt these instructions accordingly.
+A Pip "user install" will install pyChapel under ``$HOME/.local``, so make sure that
+``$HOME/.local/bin`` is near the front of your PATH.
+
+Install pyChapel and dependencies from PyPi::
+
+  $ pip install --user "numpy>=1.14" "future>=0.15.2" pyChapel
+
+Run the pyChapel self-check. Note the "Missing Chapel lib" dir location::
+
+  $ pych -k     # if "command not found", make sure $HOME/.local/bin is in PATH
+    Checking installation...
+    * Templates
+    * Object Storage
+    * Libraries
+    ERR: Missing Chapel lib file(libchpl.a) in dir(/home/chapeluser/.local/share/pych/lib)
+    ERR: Missing Chapel lib file(main.o) in dir(/home/chapeluser/.local/share/pych/lib)
+    * Commands
+
+PyChapel requires the Chapel compiler and runtime, which must be built from source
+to work with pyChapel. 
+
+Get Chapel source from Github::
+
+  $ cd $HOME    # For example- any location that will not get accidentally deleted
+  $ git clone git@github.com:chapel-lang/chapel.git
+
+Note the location::
+
+    $HOME/chapel  (in this example)
 
-Depending on your system the commands might be slightly different but the 
-should provide the gist of what is required.
+Add the following to your shell environment::
 
-pyChapel depends on the pyChapel Python module itself which is available via
-PyPi. Additionally the Chapel compiler and a library compiled with the compiler
-is required.
+    (.bashrc, for example)
 
-Installing pyChapel::
+    #
+    # Setup Chapel environment
+    #
+    # CHPL_HOME=Chapel source location
+    #
+    export CHPL_HOME=$HOME/chapel   # in this example
+    export CHPL_LIBMODE=shared
+    source $CHPL_HOME/util/quickstart/setchplenv.bash
+    #
+    # Add Pip/Python "user install" bin to PATH, if needed
+    #
+    export PATH=$HOME/.local/bin:$PATH
 
-  pip install numpy argparse pyChapel
+Build the Chapel compiler and run-time system::
 
-Build and install Chapel from source::
+    (Make sure the above shell environment has been set up first.)
 
-  cd ~
-  git clone git@github.com:chapel-lang/chapel.git
+  $ cd $CHPL_HOME
+  $ make
 
-Add this or equivalent to your environment::
+See `Quickstart <https://chapel-lang.org/docs/latest/usingchapel/QUICKSTART.html>`_ for more
+about building Chapel.
```

Comment:
nit-picky: "... for more information about building Chapel"

Reply:
okay